qa/packaging/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -96,11 +96,10 @@
 /**
  * This class tests the Elasticsearch Docker images. We have several:
  * <ul>
- *     <li>The default image with a custom, small base image</li>
- *     <li>A UBI-based image</li>
+ *     <li>The default image UBI-based image</li>
  *     <li>Another UBI image for Iron Bank</li>
  *     <li>A WOLFI-based image</li>
- *     <li>Images for Cloud</li>
+ *     <li>Image for Cloud ESS</li>
```

Comment:
its a mess. but we should just name that "Image for Cloud" sorry

Reply:
Changed